interface/schema.graphql

```diff
@@ -29,3 +29,9 @@ type MaybeUriOrManifest {
   """
   manifest: Bytes
 }
+
+type UriResolutionResult {
```

Comment:
For better backward compatibility, I'd suggest the following:
```graphql
type MaybeUriOrManifest  {
  uri: String
  manifest: String
  debug: String
}
```

This way you can write the calling logic as:
```typescript
const result = client.invoke<MaybeUriOrManifest>(...);

if (!result.ok) {
  // wrapper threw an error
  result.error
}

if (!result.value.uri && !result.value.manifest) {
  // resolution failed, look for debug info
  resolutionContext.addDebugInfo(
    result.value.debug
  );
}
```